nss-day-cohort-48/giffygram-gopher
# DM Pen & Buttons functionality

#### Changes Made
1. Modified file `MessageForm.js` to include `boolean true or false` functionality to show or hide entry form
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout anm-hiddenDirectMessage
    ```
2. Refresh server

3. Test app functionality.
    > On page load BOTH forms should be hidden
    > Clicking the PEN will show DM form, hitting cancel and 'x' will hide form
    > New giffy form functionality should be intact
    > Submitting new DM won't activate new giffy entry form to show

4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
